.github/workflows/ci_check_pr_comment.yml

```diff
@@ -32,15 +32,17 @@ jobs:
   rebase_pr:
     runs-on: [ ubuntu-latest ]
     # This job only runs for pull request comments and when the comment starts with `/rebase`
-    if: contains(github.event.comment.html_url, '/pull/') && startsWith(github.event.comment.body, '/rebase')
+    if: contains(github.event.comment.html_url, '/pull/') && startsWith(github.event.comment.body, '/rebase') && github.event.comment.author_association == 'MEMBER'
     steps:
       - uses: actions/checkout@v2
         with:
+          token: ${{ secrets.PAT }}
+          persist-credentials: false
           fetch-depth: 0
       - name: Rebase
         uses: cirrus-actions/rebase@1.4
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_ACCESS_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.PAT }}
```

Comment:
We already have a personal access token secret `GITHUB_ACCESS_TOKEN`, which we added for the code formatting checks (iirc). Is that not usable for this check?

Reply:
Oh, if it's already a personal access token then it should work. I thought it was a different one. Let me replace it with that.

Reply:
Also, please avoid using abbreviations like `PAT` and use the full name like `PERSONAL_ACCESS_TOKEN`. Someone looking at the list of secrets a month from now will have no idea what `PAT` actually is.